include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -163,12 +164,9 @@ struct __subgroup_radix_sort
                 auto __counter_lacc = __buf_count.get_acc(__cgh);
 
                 __cgh.parallel_for<_Name...>(
-                    __range, ([=](sycl::nd_item<1> __it) [[_ONEDPL_SYCL_REQD_SUB_GROUP_SIZE_IF_SUPPORTED(16)]] {
-                        union __storage
-                        {
-                            _ValT __v[__block_size];
-                            __storage() {}
-                        } __values;
+                    __range,
+                    ([=](sycl::nd_item<1> __it)[[_ONEDPL_SYCL_REQD_SUB_GROUP_SIZE_IF_SUPPORTED(__req_sub_group_size)]] {
```

Comment:
My impression was that term _req_sub_group_size_ is already included in "_ONEDPL_SYCL_REQD_SUB_GROUP_SIZE_IF_SUPPORTED" name, therefore redundant.

But of course `[[maybe-unused]]` also should work, so I'm OK with the change.

Reply:
Yep, you are right. Let's then keep 16 as a literal.